configure.ac

```diff
@@ -352,6 +352,26 @@ else
     AC_MSG_RESULT([yes])
 fi
 
+# Experimental Verbs Flow Tag
+#
+AC_ARG_ENABLE(flow_tag,
+	AC_HELP_STRING([--disable-flow-tag], [Disable Flow Tagging]),
+	[with_flow_tag=no],
+	[with_flow_tag=yes])
+
+if test "x${with_flow_tag}" = "xyes" ; then
+	AC_COMPILE_CHECK([VERBS Flow Tag check],
+	[#include <infiniband/verbs_exp.h>],
+	[struct ibv_exp_flow_spec_action_tag tag; tag = tag;],
+	ac_cv_libverbs_flow_tag=yes,
+	ac_cv_libverbs_flow_tag=no)
+	AC_MSG_RESULT($ac_cv_libverbs_flow_tag)
+	if test $ac_cv_libverbs_flow_tag = yes ; then
+		AC_DEFINE([FLOW_TAG_ENABLE], [1], [Flow Tagging enable])
```

Comment:
Please follow the rule on prefixes for definition of experimental VERBS API, see below in the file, should be like IBV_EXP_*. AC_DEFINE looks should have additional prefix DEFINED_